disnake/member.py

```diff
@@ -340,6 +341,7 @@ def __init__(
         self.nick: Optional[str] = data.get("nick")
         self.pending: bool = data.get("pending", False)
         self._avatar: Optional[str] = data.get("avatar")
+        self._banner: Optional[str] = data.get("banner")
```

Comment:
Updates of this field aren't being applied yet, like e.g. `_avatar` in `_update` and `_copy`.